get_matching_values/getmatchingvalue.js

```diff
@@ -0,0 +1,25 @@
+// receive 2 array. 
+// return matching value. if no match return null
+function getMatchingValue(a, b) {
```

Comment:
a and b are fine, but since they are arrays, I would have gone with arr1, arr2 (which also explain their type).